django_app/tests/test_views.py

```diff
@@ -1,8 +1,38 @@
 import pytest
+from django.conf import settings
 
 
 @pytest.mark.django_db
 def test_declaration_view_get(peter_rabbit, client):
     client.force_login(peter_rabbit)
     response = client.get("/")
     assert response.status_code == 200, response.status_code
+
+
+def count_s3_objects(s3_client):
```

Comment:
```suggestion
def count_s3_objects(s3_client) -> int:
```

Reply:
`count_s3_objects` is limited to 1,000 objects, could this be a problem?

Reply:
Not sure if it would be for testing on GitHub, but I guess it could be running locally. I'll look into the paginated results